libs/pong/index.js

```diff
@@ -0,0 +1,6 @@
+export * from "./coding.js";
```

Comment:
For the GCP function we would need a good `index.d.ts` file, but it might be easier to create it then.